docs/dev.rst

```diff
@@ -0,0 +1,315 @@
+###########
+Development
+###########
+
+At the time of writing, **Marsha** is developed with **Python** 3.6 for **Django 2.0**.
+
+************
+Code quality
+************
+
+We enforce good code by using some linters and auto code formatting tools.
+
+To run all linters at once, simply run:
+
+.. code-block:: shell
+
+    make lint
+
+You can also run each linter one by one. We have ones for the following tools:
+
+.. _section-black:
+
+black
+=====
+
+We use `black <https://github.com/ambv/black>`_ to automatically format python files to produce
+`pep 8 <https://www.python.org/dev/peps/pep-0008/>`_ compliant code.
+
+The best is to
+`configure your editor to automatically update the files when saved <https://github.com/ambv/black#editor-integration>`_.
+
+If you want to do this manually, simply run:
+
+.. code-block:: shell
+
+    make black
+
+And to simply check if all is correct without actually modifying the files:
+
+.. code-block:: shell
+
+    make check-black
+
+.. _section-flake8:
+
+flake8
+======
+
+In addition to :ref:`section-black` auto-formatting, we pass the code through
+`flake8 <http://flake8.pycqa.org/en/latest/>`_ to check for a lot of rules. In addition to the default ``flake8``
+rules, we use these plugins:
+
+- `flake8-bugbear <https://pypi.org/project/flake8-bugbear/>`_ to find likely bugs and design problems.
+- `flake8-comprehensions <https://pypi.org/project/flake8-comprehensions/>`_ to helps write better list/set/dict comprehensions.
+- `flake8-imports <https://pypi.org/project/flake8-imports/>`_ to check imports order via ``isort``.
+- `flake8-mypy <https://pypi.org/project/flake8-mypy/>`_ to check typing inconsistencies via ``mypy`` (see :ref:`section-mypy`).
+- `flake8-docstrings <https://pypi.org/project/flake8-docstrings/>`_ to check docstrings (see :ref:`section-docstrings`)
+
+To check your code, simply run the command:
```

Comment:
Some say that some words such as "_simple_" or "_simply_" must be avoided in a technical documentation as it may be offending to some people working hard to understand what's not so obvious.